schedule/yast/sle/flows/default_s390x_kvm.yaml

```diff
@@ -0,0 +1,61 @@
+---
+# Default ordered sequence of steps to be optionally overwritten for s390x
+bootloader:
+  - installation/bootloader_start
+setup_libyui:
+  - installation/setup_libyui
+access_beta:
+  - installation/access_beta_distribution
+product_selection:
+  - installation/product_selection/install_SLES
+license_agreement:
+  - installation/licensing/accept_license
+registration:
```

Comment:
default is to register the system, we can include this override on the yaml schedule instead of in the default file.

Reply:
changed.